src/main/java/com/jcabi/github/mock/MkNotifications.java

```diff
@@ -40,24 +42,43 @@
  *
  * @author Giang Le (lthuangiang@gmail.com)
  * @author Paul Polishchuk (ppol@ua.fm)
+ * @author Piotr Pradzynski (prondzyn@gmail.com)
  * @version $Id$
  * @since 0.15
  * @see <a href="https://developer.github.com/v3/activity/notifications/">Notifications API</a>
- * @todo #920 Implement iterate() and get() operations in MkNotifications.
- *  Don't forget about unit tests.
  * @todo #913:30min Implement markAsRead() and thread() operations in
  *  MkNotifications. Don't forget about unit tests.
  */
 @Immutable
 final class MkNotifications implements Notifications {
+
+    /**
+     * Notifications.
+     */
+    private final transient List<Notification> notifications;
+
+    /**
+     * Public ctor.
+     * @param quantity Number of notifications.
+     * @checkstyle InnerAssignmentCheck (7 lines)
+     */
+    MkNotifications(final int quantity) {
+        int identifier = 0;
+        this.notifications = Collections
+            .<Notification>nCopies(
+                quantity,
+                new MkNotification(identifier += 1)
```

Comment:
@prondzyn  I think you get `quantity` copies of `new MkNotification(1)`, right?
Let's initialize this one via `for` loop.